src/components/media/MediasSelectModal.jsx

```diff
@@ -291,16 +368,259 @@ const MediasSelectModal = ({
                 />
               </Center>
             )}
-            <LoadingButton
-              alignSelf="flex-end"
-              id="selectMedia"
-              isDisabled={!watch("selectedMedia")}
-              onClick={handleSubmit((data) => onProceed(data))}
-            >
-              Select
-            </LoadingButton>
-          </VStack>
-        </ModalFooter>
+          </div>
+        </Box>
+      </GridItem>
+    </Grid>
+  )
+
+  return (
+    <Modal
+      isOpen
+      onClose={onClose}
+      size="6xl"
+      scrollBehavior="inside"
+      closeOnOverlayClick={false}
+      isCentered
+      overflowY="hidden"
+    >
+      <ModalOverlay />
+      <ModalContent
+        padding="0.5rem"
+        paddingTop="1rem"
+        maxHeight="90%"
+        overflowY="hidden"
+      >
+        {mediaType === "images" && allowExternal ? (
```

Comment:
maybe we should just make 2 modals? might help to simplify this. just offering suggestions, am ok with this atm as functionality is ok and it's not _super_ complicated.

Reply:
agreed!